tests/testCameraMapper.py

```diff
@@ -413,16 +439,19 @@ def test(self):
         registryA = butler._repos.inputs()[0].repo._mapper.registry
         registryB = butler._repos.outputs()[0].repo._mapper.registry
         self.assertEqual(id(registryA), id(registryB))
+        del registryA
+        del registryB
         del butler
 
-        with open(os.path.join(repoBRoot, 'registry.sqlite3'), 'w') as f:
-            f.write('123')
+        self._createRegistry(os.path.join(repoBRoot, 'registry.sqlite3'))
         butler = dafPersist.Butler(inputs=self.repoARoot, outputs=repoBRoot)
         # see above; don't copy this way of getting the registry.
         registryA = butler._repos.inputs()[0].repo._mapper.registry
         registryB = butler._repos.outputs()[0].repo._mapper.registry
         self.assertNotEqual(id(registryA), id(registryB))
-
+        del registryA
+        del registryB
+        del butler
```

Comment:
Do these make a difference now? The variables will disappear at end of scope and the `gc.collect()` will clean up.